ui/app/scripts/task/views/executions.html

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+	<div class="col-md-12">
+		<div class="col-md-12 table-filter">
+			<div class="col-md-4 col-md-offset-8">
+				<input type="text" class="form-control" ng-model="filterQuery" id="filterTable" placeholder="Quick filter">
+			</div>
+		</div>
+	</div>
+</div>
+<table class="table table-striped table-hover">
+	<thead>
+		<tr>
+			<th>Execution Id</th><th>Task Name</th><th>Start Time</th>
+			<th>End Time</th><th>Exit Code</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr dir-paginate="item in pageable.items | itemsPerPage: pageable.pageSize | filter:filterQuery" total-items="pageable.total"
+		    ng-show="!item.inactive">
+			<td>{{item.executionId}}</td>
+			<td>{{item.taskName}}</td>
+			<td xd-date-time="{{item.startTime}}"></td>
+			<td xd-date-time="{{item.endTime}}"></td>
+			<td>{{item.exitCode}}</td>
```

Comment:
Given we have a story https://github.com/spring-cloud/spring-cloud-dataflow-ui/issues/22 to track executions tab, may be the entire page implementation is the scope of that story.